go/controller/internal/controller/prautomation_controller.go

```diff
@@ -173,6 +173,10 @@ func (in *PrAutomationReconciler) sync(ctx context.Context, prAutomation *v1alph
 	if err := in.ensure(prAutomation); err != nil {
```

Comment:
Shouldn't we just move this to above line 155?  We technically call `in.attributes(...)` twice

Reply:
should be fine, done